index.js

```diff
@@ -0,0 +1,41 @@
+const map = (array, callback) => {
+    let newArray = [];
+
+    for (const item of array) {
+        const newItem = callback(item);
+        newArray = [...newArray, newItem]
+    }
+    return newArray;
+};
+
+const filter = (array, callback) => {
+    let newArray = [];
+
+    for (const item of array) {
+        const newItem = callback(item);
+        if (newItem === true) {
```

Comment:
This works great! We can also shorten the condition to the shorthand of `if(newItem) {...}` :)